python/shotgun_menus/entity_field_menu.py

```diff
@@ -164,6 +166,10 @@ def set_entity_type_filter(self, entity_type_filter):
         """
         self._entity_type_filter = entity_type_filter
 
+    # this was causing RV to segfault on exit. i think
+    # the task manager was already freed in another thread
+    # before this happens.
+    # removed the registration above and the crashing stopped.
```

Comment:
Same as above.  We should either figure out the crash with the registration or remove it and this comment.